File: FE.Core.FW/src/apis/models/TimeFrameInDayModel.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

export type TimeFrameInDayModel = {
  id?: string;
  sysTimeFrameId?: string;
  maxRegistry?: number;
  timeStart?: string | null;
  timeEnd?: string | null;
  isShow?: boolean;
};

export type TimeFrameInDayModelCustom = {
  id?: string;
  sysTimeFrameName?: string;
  maxRegistry?: number;
  timeStart?: string | null;
  timeEnd?: string | null;
  isShow?: boolean;
};

File: BE.Core.FW/Backend/Business/TimeFrameInDay/TimeFrameInDayModel.cs
using System.ComponentModel.DataAnnotations;

namespace Backend.Business.TimeFrameInDay
{
    public class TimeFrameInDayModel
    {
        public Guid Id { get; set; }
        public Guid SysTimeFrameId { get; set; }
        public int MaxRegistry { get; set; }
        public string TimeStart { get; set; } = string.Empty;
        public string TimeEnd { get; set; } = string.Empty;
        public bool IsShow { get; set; } = true;
    }
}
